src/nat/cli/commands/workflow/workflow_commands.py

```
@@ -122,235 +95,200 @@ def find_package_root(package_name: str) -> Path | None:
-        install (bool): Whether to install the workflow package immediately.
-        workflow_dir (str): The directory to create the workflow package.
-        description (str): Description to pre-popluate the workflow docstring.
-    """
+@click.argument("workflow_name")
+@click.option("--install/--no-install", default=True, help="Install workflow after generation.")
+@click.option("--workflow-dir", default=".", help="Output directory for the workflow project.")
+@click.option("--description", default="NAT function template. Please update the description.")
+def create_command(workflow_name: str, install: bool, workflow_dir: str, description: str) -> None:
+    """Create a new workflow project from templates."""
+    if not workflow_name or not workflow_name.strip():
+        raise click.BadParameter("Workflow name cannot be empty.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)